gradio/analytics.py

```diff
@@ -224,7 +224,7 @@ def get_inputs_outputs(
             return [b.get_block_name() for b in components] if components else None
         return fallback
 
-    core_components = core_gradio_components()
+    core_components = [get_block_name(c) for c in core_gradio_components()]
```

Comment:
btw we're using analytics.get_block_name here, but we're using Block.get_block_name above. They are not the same function, which might lead to false positives

Reply:
I think this is ok because `Block.get_block_name` is a subset of `analytics.get_block_name`. In `analytics.get_block_name` we don't map the template class name to its parent, e.g. `Text` stays `text` and not `textbox`.

Reply:
Ah right 👍